### ISSUE TITLE
10x loss scale difference in Flux sft

### ISSUE THREAD

**Issue by ssj9596** [NVlabs/FastGen#11]
Hi team,

I'm doing sft on Flux.1 and noticed a consistent ~10× difference in reported training loss:

My custom framework: loss in 0.0 ~ 0.1 range
FastGen framework: loss in 0.0 ~ 1.0 range (exactly ~10× higher)

I double checked both codebases, and there is no explicit loss scaling

Both frameworks converge well and the trained models work

But why is the loss scale different? Do you know of any reasons or have any insights on this?

Thanks!
